runtime/web_worker.rs

```diff
@@ -524,132 +519,3 @@ pub fn run_web_worker(
   debug!("Worker thread shuts down {}", &name);
   result
 }
-
-#[cfg(test)]
-mod tests {
-  use super::*;
-  use crate::tokio_util;
-
-  fn create_test_web_worker() -> WebWorker {
-    let main_module = deno_core::resolve_url_or_path("./hello.js").unwrap();
```

Comment:
Why remove these tests?

Reply:
Because the test can not be implemented in the new design without a reference to the parent workers' OpState, but there is no parent worker in these tests. The scenarios are all tested via js unit tests already.